www/src/scss/viewer/verse-slide/_verse-slide.scss

```diff
@@ -51,6 +51,14 @@
   font-weight: 600;
   margin: 10px 0;
   order: 2;
+	&.custom-english {
+		display: flex;
+		flex-direction: column;
+		flex-wrap: wrap;
+		height: 100%;
+		overflow-y: auto;
```

Comment:
I noticed the Ardaas Overview slide contains horizontal scroll. Kudos to the solution veerji. 

However, a situation came in my mind about the chromecast screen. We usually don't get access of mouse on most of the chromecast screens we use for Bani casting. Even if we give a vertical scroll this problem will persist. What do you think we can do in this case? Ideally the verse should be divided in multiple parts. But what do you think veerji? 

![Uploading Peek 2022-10-26 20-39.gif…]()

Reply:
@Gauravjeetsingh veerji, should I open issue for this beforehand?

Reply:
Go ahead